Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -317,43 +275,11 @@ $ cat entitlements.plist
 
 Note the application identifier, which is a combination of the Team ID (LRUD9L355Y) and Bundle ID (sg.vantagepoint.repackage). This provisioning profile is only valid for the app that has this App ID. The `get-task-allow` key is also important: when set to `true`, other processes, such as the debugging server, are allowed to attach to the app (consequently, this would be set to `false` in a distribution profile).
 
-#### Other Preparations
```

Comment:
Can you add a link in line 222 to optool in the tools section? Then the link is not lost for this content. In line 222 (in your PR) optool is mentioned.

Reply:
done